### ISSUE TITLE
!!! Serious warning - Dangerous product please remove immediately from market !!!

### ISSUE THREAD

**Issue by ServeurpersoCom** [linshuqin329/UPS-18650#2]
Need a lot of cool and safe power source for small robots on our robotics cloud (zero ads https://www.vigibot.com), this is why we choose 18650 (manganese oxide) way safer than LiPoly.
I mounted two USP units at a time before dozens of others more for this project.

First, design is amazingly beautiful ! But it's a trap !

Result : Two different Raspberry PI 3B and 3B+ get killed on two different UPS-18650
One UPS PCB go to fire, one smoke and won't light up.
2 on 4 18650 battery are damaged (1 need a new heat-shrinkable sheath).

There is a space problem between solder and battery, it's invisible, it's a trap !
The hole made in the battery insulation shrink film is hard to see with naked eye.

Excuse my english.
(zero ads, robots dev server : https://www.serveurperso.com/?page=robots)

**Comment by ServeurpersoCom** [comment]
Please note this is the last HARDWARE REV 1.1

A product of this type need a true lithium battery controller with more technical margin, current limitations and fuse.
It need more space between dangerous shortcut point and no plastic near power component which represents a significant risk of fire in case of malfunction.

**Comment by ServeurpersoCom** [comment]
I warned amazon's sellers for the risk of fire. I can't do more. Good luck

**Comment by linshuqin329** [comment]
hi Serveurperso, firstly thank you for your advice.  and i want to make sure that  have you ever used a old 18650 battery  which is without the heat-shrinkable sheath, so that make the pin of the pi can touch with  the battery and destroy the pi。secondly，the UPS-18650 it is have a Li battery protection IC XB8089D,it can protect the battery from Overcurrent Detection Current. so i'm sorry. I'm not sure what caused the damage of your pi. you can tell me more detail about the operation of the UPS. thank you.

**Comment by ServeurpersoCom** [comment]
Hello,

It is fundamental NEVER to use a cylindrical Lithium cell without the heat-shrinkable sheath !!! it's dangerous because the housing is entirely grounded and make short circuit easier.

Your design lacks of margin space.

The UPS product here have a big problem : the raspberry Pi welds are sharp needle which pierces the 18650 envelope, this make an hidden short circuit that instant-kill the raspberry pi.

The DC-DC component used inside this product is undersized and/or poorly implemented (missing filtering) and sometime a transient current overshoot kill it / and it goes into thermal runaway & fire.

In this case we get a non current limited lithium battery >10amp shot circuit which always ends in PCB and plastic fire.

For product of this type please don't forget :

- More margin space : the air is a very good insulator @ this voltage !
- A reliable hardware current limitation in case of main DC-DC failure - a small PCB track or better: a good plain old fuse !
- No plastic near component that can catch fire!

Pascal

**Comment by ServeurpersoCom** [comment]
https://elinux.org/Polyfuses_explained

**Comment by ServeurpersoCom** [comment]
There is the same issue with the small version UPS-Like for PI Zero/w
No fire but there is not enough space between the battery and the welds, it's make short circuit between pin header of the PI Zero. And it bend the PCBs.

I unsoldered the 1000mAh LiPoly to put a thinner (800 or 900mAh ?) cell.

**Comment by acolwill** [comment]
I received my unit from AliExpress yesterday and configured it.
There is a clearance issue between the 18650 cells and the underside of the RaspberryPi.
One of the pins for the ethernet port touches an 18650 cell which can cause a direct short to ground.
No heatsink can be added to the memory module.

By increasing the height of the holder by ~2mm it would allow for a thin heatsink on the memory module and would give sufficient clearance so no pins can touch either of the 18650 cells.

**Comment by linshuqin329** [comment]
hi Serveurperso and acolwill。To solve problems of the UPS-18650, you can cut off the pins for the ethernet port like this, and the subsequent productions i will  increasing the height of the acrylic by 1mm to solve the problem. but the heatsink will not be considered.
![1](https://user-images.githubusercontent.com/33977023/46075648-ba194c80-c1bd-11e8-833e-6f4249cb5829.jpg)
![img_20171127_150135_ _](https://user-images.githubusercontent.com/33977023/46075654-bede0080-c1bd-11e8-8e2e-cca0b6a2bd67.jpg)

**Comment by acolwill** [comment]
Thank you for your prompt response!

Additionally, I was monitoring the voltages last night and was concerned because the Rpi was reporting cell voltages of 4.22v (confirmed with DMM).
This has since dropped to 4.21v however the monitoring script is still reporting 103% capacity??

**Comment by linshuqin329** [comment]
that is ok ,with a few of the cycles of charge and discharge, the value of the capacity will be more accurate. of course,if you want to get the most accurate value of the capacity, you should program the  battery configuration data into the IC. for more detail,you can read from the datasheet of the MAX17040
![111](https://user-images.githubusercontent.com/33977023/46077774-a4f3ec00-c1c4-11e8-8bc1-dd9f22809d56.JPG)
![2222](https://user-images.githubusercontent.com/33977023/46077777-a58c8280-c1c4-11e8-812a-39b14ac5d9ee.JPG)

**Comment by ServeurpersoCom** [comment]
I agree heatsink (and overclocking) is not usefull on battery operated PI.

Remember I killed two UPS-18650 and two PI3B+ because I assembled UPS with PI like a work in chain for robots and I was missed any verification because of the splendid PCB design. I also discovered that just cut 1 ethernet pin, but too late.

How to get one sample from the new batch ? Do you solve the same problem with UPS-Lite for PI Zero/w also ? Do you add the polyfuse like some other products ?

**Comment by linshuqin329** [comment]
i think that it is not a problem with  UPS-Lite ，the Pin Headers are just can touch the battery ，but cannot impale the battery。 it is safety。and about the polyfuse ，i have test two kinds of polyfuse ，the protection current is 3.5A，but the trip current was  too high，and with the large load current, the polyfuse  will cause a few voltage drop ,that will make the output voltage under 5V. so i think that the polyfuse is not a good idea. 
![_20180926231454](https://user-images.githubusercontent.com/33977023/46091172-d977a000-c1e4-11e8-84ab-9bab7629a0f8.jpg)

**Comment by ServeurpersoCom** [comment]
For UPS lite also you must add few mm because it's not pedantic to get pinch on lipo.
I buyed a lot of different UPS, the only one with beautiful PCB is your one, all other look crap & low quality. solder pin who marks the battery is bad.

Also you can not control the manufacturing variations between units (glue?) and the 1000mAh cell from another supplier can be a bit larger. LiPo must be easy to replace.

Also for 18650 version, for casing you must cut a large area where the heat is.

**Comment by ServeurpersoCom** [comment]
I get other product with a 3A polyfuse and there is no voltage drop problem @ full load on RPI 3B+

**Comment by yuret** [comment]
Dear author, are you planning to release next version of you UPS? If so can you tell me when? Also, can you tell what IC is used to charge batteries? Thank you.

**Comment by runout0** [comment]
I agree to the TO (Serveurperso).
I use a Raspi Pi 2 Model B.
The Pins of the Network-Jack (RJ45) colliding with the Battery!

There is a nearly invisible hole in the insualtion.

Kind Regards
Runout

**Comment by sprlightning** [comment]
Maybe you can try removing the battery base and welding it to the back of the PCB so that no pins pierce the battery skin.
